lib/testutils/cleanEtcd.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2016 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testutils
+
+import (
+	"log"
+	"os/exec"
+)
+
+// CleanEtcd is a utility function to wipe clean "/calico" recursively from etcd.
+func CleanEtcd() {
+	err := exec.Command("etcdctl", "rm", "/calico", "--recursive").Run()
```

Comment:
this places another requirement on running the tests no?  Is the user going to have etcdctl installed?  Spinning up etcd is handled through a container which is more reasonable, but requiring etcdctl to be installed seems more worrying.

Can we just change this to be a curl request - since we can probably do that using golang native library stuff?

How does CNI do this, or does it also use etcdctl?